code/application/weekly-ensemble/build_ensembles.R

```diff
@@ -20,13 +20,19 @@ for (root in save_roots) {
   if (!file.exists(root)) dir.create(root, recursive = TRUE)
   if (!file.exists(paste0(root,"ensemble-metadata/"))) {
   dir.create(paste0(root,"ensemble-metadata/"), recursive = TRUE)
-}
+  }
 }
 
 # Where to save plots
 plots_root <- 'code/application/weekly-ensemble/plots/COVIDhub-ensemble/'
 if (!file.exists(plots_root)) dir.create(plots_root, recursive = TRUE)
 
+# Where to save hospitalization exclusion tables
+sd_check_table_path <- 'code/application/weekly-ensemble/exclusion-outputs/tables/'
+if (!file.exists(sd_check_table_path)) dir.create(sd_check_table_path, recursive = TRUE)
+sd_check_plots_path <- 'code/application/weekly-ensemble/exclusion-outputs/plots/'
+if (!file.exists(sd_check_plots_path)) dir.create(sd_check_plots_path, recursive = TRUE)
```

Comment:
Suggesting to change `sd_check_plots_path` to `sd_check_plot_path` here -- I got an error further down when `sd_check_plot_path` was referenced.
```suggestion
sd_check_plot_path <- 'code/application/weekly-ensemble/exclusion-outputs/plots/'
if (!file.exists(sd_check_plot_path)) dir.create(sd_check_plot_path, recursive = TRUE)
```